.circleci/config.yml

```diff
@@ -402,3 +473,9 @@ workflows:
       - cpp_lint
       - js_lint
       - install_and_test_ubuntu
+      - update_docs:
+          requires:
+            - install_and_test_ubuntu
+          filters:
+            branches:
+              only: autodoc
```

Comment:
```suggestion
              only: master
```

?

Reply:
Yes, planned to change to master, still need for testing.